Sources/Purchasing/StoreKit2/SK2AppTransaction.swift

```diff
@@ -18,13 +18,15 @@ import StoreKit
 internal struct SK2AppTransaction {
 
     @available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *)
-    init(appTransaction: AppTransaction) {
+    init(appTransaction: AppTransaction, jwsRepresentation: String) {
         self.bundleId = appTransaction.bundleID
         self.originalApplicationVersion = appTransaction.originalAppVersion
         self.originalPurchaseDate = appTransaction.originalPurchaseDate
         self.environment = .init(environment: appTransaction.environment)
+        self.jwsRepresentation = jwsRepresentation
```

Comment:
This is similar to what we already so with the `SK2StoreTransaction`. The JWS token is part of the `VerificationResult`, not the `AppTansaction`, but we want to preserve it.